server/moodle/mod/livequiz/classes/answers/answers.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+namespace mod_livequiz\answers;
+
+use dml_exception;
+use dml_transaction_exception;
+use mod_livequiz\questions_answers_relation\questions_answers_relation;
+use mod_livequiz\quiz_questions_relation\quiz_questions_relation;
+
+class answers
+{
+    private $correct;
+    private $description;
+    private $explanation;
```

Comment:
Why are the answer id not a part of the object? While you should not be able to set or change it, I think it is worth keeping track of